commons/src/main/java/org/infinispan/commons/marshall/ProtoStreamTypeIds.java

```diff
@@ -78,10 +78,12 @@
    int FILE_READ_LOCK_KEY = LUCENE_LOWER_BOUND + 4;
    int FILE_LIST_CACHE_VALUE = LUCENE_LOWER_BOUND + 5;
 
-   // Scripting 4800 -> 4999
+   // Tasks + Scripting 4800 -> 4999
    int SCRIPTING_LOWER_BOUND = LUCENE_LOWER_BOUND + 200;
    int EXECUTION_MODE = SCRIPTING_LOWER_BOUND;
    int SCRIPT_METADATA = SCRIPTING_LOWER_BOUND + 1;
+   int DISTRIBUTED_SERVER_TASK = SCRIPT_METADATA + 1;
+   int DISTRIBUTED_SERVER_TASK_PARAMETER = DISTRIBUTED_SERVER_TASK + 1;
```

Comment:
This is much simpler than +5 etc. I'll update the class to use this approach for all values in the future.

Reply:
You just can't comment them out arbitrarily anymore though :D